### PR TITLE
Fix creation of release PR in release workflow. 

### PR DESCRIPTION
**Pull request by mpeddada1**
To be able to trigger unit tests from the release PR.

### CODE REVIEW

File: .github/workflows/jib-cli-release.yml [hunk 1]
```diff
@@ -46,8 +46,8 @@ jobs:
       - name: Create pull request
         uses: repo-sync/pull-request@v2
         with:
-          # Provided by Actions. No need to create a token.
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          # Use a personal access token instead of the one created by Github Actions.
```
Comment (chanseokoh) [inline .github/workflows/jib-cli-release.yml:49]: Something like

```
# Use a personal token to file a PR as a non-bot author to trigger other workflows (e.g., unit tests):
# <link to the official doc about why using a personal token>;
```

File: .github/workflows/jib-cli-release.yml [hunk 2]
```diff
@@ -46,8 +46,8 @@ jobs:
...
+          github_token: ${{ secrets.GA_RELEASE_PR_PERSONAL_TOKEN }}
           source_branch: cli-release-v${{ github.event.inputs.release_version }}
           pr_title: "CLI release v${{ github.event.inputs.release_version }}"
           pr_body: "To be merged after the release is complete."
```
Comment (chanseokoh) [inline .github/workflows/jib-cli-release.yml:53]: Remove the kokoro label below.

**Review by chanseokoh — APPROVED** [review]

File: .github/workflows/jib-cli-release.yml (hunk 2 above)
Comment (chanseokoh) [reply .github/workflows/jib-cli-release.yml:54]: Oh, we should also add `PR: Merge After Release` too.

Comment (mpeddada1) [reply .github/workflows/jib-cli-release.yml:54]: oops nice catch

**Pull request merged**